backend/canvas_app_explorer/serializers.py

```diff
@@ -10,7 +14,16 @@ class Meta:
 class LtiToolSerializer(serializers.ModelSerializer):
     # Addiitonal expanded ead only field to make it easier on the front-end
     canvas_placement_expanded = CanvasPlacementSerializer(read_only=True, many=True, source="canvas_placement")
+    enabled = serializers.SerializerMethodField()
 
     class Meta:
         model = models.LtiTool
         fields = '__all__'
+
+    def get_enabled(self, obj: models.LtiTool) -> Union[bool, None]:
+        available_tools = self.context['available_tools']
+        matches: List[ExternalTool] = list(filter(lambda x: x.id == obj.canvas_id, available_tools))
```

Comment:
Could you leave a short code comment before this line? I'm not sure to describe what this all is doing here.

Reply:
I can add that, yeah. It's basically doing a search for the tool in the available tools by canvas ID.

Reply:
Done. See https://github.com/tl-its-umich-edu/canvas-app-explorer/pull/167/commits/f0237d97daddae246615d8da71c40422eeaa8ea5